LICENSE

```diff
@@ -1,6 +1,6 @@
-The MIT License (MIT)
+MIT License
 
-Copyright (c) 2021 Plotly, Inc
+Copyright (c) 2016-2024 Plotly Technologies Inc.
```

Comment:
@alexcjohnson 
The start was in 2012 or earlier?

Reply:
There's probably a nugget or two in here from 2012, yes. As discussed with @gvwilson though, I'd advocate for removing these years altogether. Seems like a relatively common practice at this point, that rids us of this kind of housekeeping permanently.